app/components/Settings/Scheduler/createModal.jsx

```diff
@@ -0,0 +1,233 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import Select from 'react-select';
+import { Controlled as CodeMirror } from 'react-codemirror2';
+
+import { Row, Column } from '../../layout.jsx';
+import Modal from '../../modal.jsx';
+import { mapDialect } from './util.js';
+
+function noop() {}
+
+const FREQUENCIES = [
+    { label: 'Run every minute', value: 60 },
+    { label: 'Run every 5 minutes', value: 5 * 60 },
+    { label: 'Run hourly', value: 60 * 60 },
+    { label: 'Run daily', value: 24 * 60 * 60 },
+    { label: 'Run weekly', value: 7 * 24 * 60 * 60 }
+];
+
+const styles = {
```

Comment:
Not a blocking change, but these styles should go into its own file, e.g. `create-modal.css`; (having so many `<xxxx style={styles.xxxx}>` is a maintenance burden).